pynxtools_xps/reader_utils.py

```diff
@@ -286,7 +286,7 @@ def construct_detector_data_key(spectrum):
     if "scan_no" in spectrum:
         scan_key = f'scans/Scan_{spectrum["scan_no"]}'
     else:
-        scan_key = "scans/Scan_0"
+        scan_key = f"scans/Scan_0"
```

Comment:
```suggestion
        scan_key = "scans/Scan_0"
```

Reply:
Nothing is inserted here, so the f-string is not necessary

Reply:
removed